editor/src/components/canvas/canvas-utils.ts

```diff
@@ -1940,26 +1922,26 @@ export function moveTemplate(
 
               const newSelectedViews = selectedViews.map((v) => {
                 if (TP.pathsEqual(v, target)) {
-                  return newInstancePath
+                  return newPath
                 } else {
                   return v
                 }
               })
 
               const newHighlightedViews =
                 newParentPath == null
-                  ? highlightedViews.map((t) => (TP.pathsEqual(t, target) ? newInstancePath : t))
+                  ? highlightedViews.map((t) => (TP.pathsEqual(t, target) ? newPath : t))
                   : [newParentPath]
 
               const updatedEditorState: EditorState = {
                 ...workingEditorState,
-                selectedViews: filterMultiSelectScenes(Utils.stripNulls(newSelectedViews)),
```

Comment:
is this a change in behavior? I don't remember why we filtered multiselected scenes: was it because there was a technical problem with them, or to prevent the user from doing something potentially weird? I _think_ this was just a technicality but I don't remember

Reply:
It was an old world technicality based on the fact that interactions made whilst a scene was selected would either change the root view of the component rendered by the scene, or would change the json object that used to be at the top of the old `ui.jsx` file :D